code/game/objects/items/melee/misc.dm

```diff
@@ -191,10 +233,9 @@
 	. = ..()
 	if(iscarbon(target))
 		var/mob/living/carbon/H = target
-		var/loss = H.getStaminaLoss()
 		H.Dizzy(10)
 		H.adjustStaminaLoss(30)
-		if((loss > 40) && prob(loss)) // if above 40, roll for sleep using 1% every 1 stamina damage
+		if(CHECK_STAMCRIT(H) != NOT_STAMCRIT)
```

Comment:
Just gonna leave a little note here to say I asked kev to change this okies thankies -w-